relayer/src/chain/mock.rs

```diff
@@ -426,13 +417,20 @@ impl ChainEndpoint for MockChain {
     fn query_consensus_state(
         &self,
         request: QueryConsensusStateRequest,
-    ) -> Result<AnyConsensusState, Error> {
+        include_proof: IncludeProof,
+    ) -> Result<(AnyConsensusState, Option<MerkleProof>), Error> {
+        // IncludeProof::Yes not implemented
+        assert!(matches!(include_proof, IncludeProof::No));
+
         let consensus_states = self.context.consensus_states(&request.client_id);
-        Ok(consensus_states
-            .into_iter()
-            .find(|s| s.height == request.consensus_height)
-            .unwrap()
-            .consensus_state)
+        Ok((
+            consensus_states
```

Comment:
Can we pull this expression out into a variable to improve readability?

Reply:
[54c8127](https://github.com/informalsystems/ibc-rs/pull/2226/commits/54c8127586d33156b302d979a68984046999031a)